src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/AmendCaseConsentedMidHandlerTest.java

```diff
@@ -0,0 +1,61 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.handler;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import uk.gov.hmcts.reform.ccd.client.model.CallbackRequest;
+import uk.gov.hmcts.reform.finrem.caseorchestration.ccd.callback.CallbackType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.mapper.FinremCaseDetailsMapper;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseData;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.FinremCaseDetails;
+import uk.gov.hmcts.reform.finrem.caseorchestration.service.ConsentOrderService;
+
+import static org.mockito.Mockito.verify;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.TestConstants.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.EventType.AMEND_CASE;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseType.CONSENTED;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.test.Assertions.assertCanHandle;
+
+@ExtendWith(MockitoExtension.class)
+class AmendCaseConsentedMidHandlerTest {
+
+    private AmendCaseConsentedMidHandler handler;
+    @Mock
+    private ConsentOrderService consentOrderService;
+
+    @Mock
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    void setUp() {
+        FinremCaseDetailsMapper finremCaseDetailsMapper = new FinremCaseDetailsMapper(objectMapper);
+        handler = new AmendCaseConsentedMidHandler(finremCaseDetailsMapper, consentOrderService, objectMapper);
+    }
+
+    @Test
+    void canHandle() {
+        assertCanHandle(handler, CallbackType.MID_EVENT, CONSENTED, EventType.AMEND_CASE);
+    }
+
+    @Test
+    void givenCase_whenCaseNatureOfApplicaftionUpdated_handleMidEvent() {
```

Comment:
The way to test this handler is to check that it returns errors.
So you'd
1. Mock ConsentOrderService to return a list of errors
2. Test that the response from the handler contains the errors

Reply:
Mocked to test for errors from the ConsentOrderService